cogs/pug.py

```diff
@@ -1108,11 +1253,42 @@ async def adminremove(self, ctx, player: discord.Member):
         if self.pugInfo.removePlayerFromPug(player):
             await ctx.send('**{0}** was removed by an admin. {1}'.format(display_name(player), self.pugInfo.format_pug()))
 
+    @commands.command(aliases=['setserver','setactiveserver'])
+    @commands.has_permissions(manage_guild=True)
+    @commands.check(isPugInProgress_Warn)
+    async def adminsetserver(self, ctx, svindex: int):
+        """Overrides active server. Admin only"""
```

Comment:
"Sets the active server to the index chosen from the pool of available servers. Admin only"